src/dx/publish.rs

```diff
@@ -245,7 +246,7 @@ where
                 method: TransportMethod::Post,
                 query_parameters: query_params,
                 body: Some(m_vec),
-                ..Default::default()
+                headers: [(CONTENT_TYPE.into(), APPLICATION_JSON.into())].into(),
```

Comment:
Do we want to have this as a default for every request?

Reply:
I think so

Reply:
You can reimplement `Default` trait by hand. 
It will be very handy to have `Default` implemented to as many our structs as possible according to API guidelines and this headers sounds like that.

Reply:
Oh... wait. For every one with payload, therefore probably not a deafult

Reply:
Ahhh. You mean messages without any payload?

Reply:
Yes. I mean there's no content type if there's no content. It makes sense to set a content type only when we actually have a content